src/dsp_tools/clients/list_client_live.py

```diff
@@ -69,3 +83,88 @@ def _reformat_children(self, list_child: list[dict[str, Any]], current_nodes: li
             current_nodes.append(OneNode(child["name"], child["id"]))
             if grand_child := child.get("children"):
                 self._reformat_children(grand_child, current_nodes)
+
+
+@dataclass
+class ListCreateClientLive(ListCreateClient):
+    api_url: str
+    project_iri: str
+    auth: AuthenticationClient
+
+    def create_new_list(self, list_info: dict[str, Any]) -> str | None:
+        url = f"{self.api_url}/admin/lists"
+        logger.debug(f"POST create new list: {list_info['name']}")
```

Comment:
The POST request is logged separately, so I think you can omit this. If you'd like to log that this method has been called, then I'd reformulate: `f"Method create_new_lists() has been called with {list_info['name']}"`